src/main/java/com/epam/reportportal/jobs/service/storage/ProjectStorageHandler.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.reportportal.jobs.service.storage;
+
+import com.epam.reportportal.jobs.repository.AttachmentRepository;
+import com.epam.reportportal.jobs.repository.ProjectRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
+ */
+@Service
+public class ProjectStorageHandler implements AllocatedStorageHandler {
```

Comment:
no need to use 3 level arch here. classes are pretty simple, so we can use everything in job class